bin/einhorn

```diff
@@ -181,6 +181,20 @@ EOF
   end
 end
 
+
+BIND_PATTERN = /\A
+  # we accept two types of socket address
+  (?:
+    # ip, host:port
+    (?:(?<host>[^:]+):(?<port>\d+)) |
+    # unix socket path
+    (?<path>[^,:]+)
+  )
+
+  # flags are optional, comma delimited, and come at the end
+  (?<flags>(?:,\w+)*)
```

Comment:
We know they would be wrong anyway. :stuck_out_tongue: 
Right now, what if they wanted to make a path with a comma or colon in it?! Spaces and unicode would work. I really don't care that much. Give em enough rope to hang themselves with!